### ISSUE TITLE
Spacing between sections should be bigger

### ISSUE THREAD

**Issue by monicacrusellasfanlo** [alphagov/govuk-design-guide#199]
### Accessibility issue or suggested improvement

We should increase the space between sections so it's easier to understand when a section ends and a new one begins. 

### Affected platform(s)

None
